deep-prove/src/bin/worker/api.rs

```diff
@@ -44,6 +44,31 @@ pub async fn serve(args: RunMode) -> anyhow::Result<()> {
     };
     crate::setup_logging(json);
 
+    #[cfg(not(feature = "disable-aws-marketplace"))]
```

Comment:
Can you please use `feature = "aws-marketplace"` rather? `not [...] disable` is the kind of think that is unpleasant to read before a morning coffee ;)